README.md

```diff
@@ -11,3 +11,154 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
```

Comment:
Please ignore my above comment @dayotte. I didn't see the change you made. That looks good now.

The only change I would suggest is to:
1. use the markdown single backquote to give code formatting to the file paths (see: https://daringfireball.net/projects/markdown/syntax#precode). 
2. Add the "./" (i.e. \`./news\`) to the path so it's clear that we're talking about the current directory.